docs/faq.rst

```diff
@@ -3,9 +3,123 @@ FAQ
++++++++++++++++++++++++++++++
+- The ``--parallel--safe-build`` CLI argument has been removed, no longer needed.
+- When you want to pass an option to a test command, e.g. to ``pytest``, now you must use ``--`` as a separator, this
+  worked with version 3 also, but any unknown trailing arguments were automatically passed through, while now this is
+  no longer the case.
+
+tox 4 - packaging changes
++++++++++++++++++++++++++
+- We use isolated builds (always) as specified by :pep:`518` and use :pep:`517` to communicate with the build backend.
+- The ``--develop`` CLI flag or the :ref:`use_develop` settings now enables editable installations via the :pep:`660`
+  mechanism rather than the legacy ``pip install -e`` behaviour. The old functionality can still be forced by setting
+  the :ref:`package` setting for the run environment to ``editable-legacy``.
+
+tox 4 -- output changes
++++++++++++++++++++++++
+- We now use colors for reporting, to help make the output easier to read for humans. This can be disabled via the
+  ``TERM=dumb`` or ``NO_COLORS=0`` environment variables, or the ``--colored no`` CLI argument.
+
+New features in tox 4
+---------------------
+Here is a non-exhaustive list of these.
+
+- You can now build wheel(s) instead of a source distribution during the packaging phase by using the ``wheel`` setting
+  for the :ref:`package` setting. If your package is a universal wheel you'll likely want to set the
+  :ref:`wheel_build_env` to ``.pkg`` to avoid building a wheel for every Python version you target.
+- Editable wheel support was added as defined by :pep:`660` via the :ref:`package` setting to ``editable``.
+- We redesigned our CLI interface, we no longer try to squeeze everything under single command, instead now we have
+  multiple sub-commands. For backwards compatibility if you do not specify a subcommand we'll assume you want the tox 3
+  legacy interface (available under the legacy subcommand), for now the list of available commands are:
+
+  .. code-block:: bash
+
+    subcommands:
+      tox command to execute (by default legacy)
+
+      {run,r,run-parallel,p,depends,de,list,l,devenv,d,config,c,quickstart,q,exec,e,legacy,le}
+        run (r)                   run environments
+        run-parallel (p)          run environments in parallel
+        depends (de)              visualize tox environment dependencies
+        list (l)                  list environments
+        devenv (d)                sets up a development environment at ENVDIR based on the tox configuration specified
+        config (c)                show tox configuration
+        quickstart (q)            Command line script to quickly create a tox config file for a Python project
+        exec (e)                  execute an arbitrary command within a tox environment
+        legacy (le)               legacy entry-point command
+
+  The ``exec`` and ``depends`` are brand new features. Other subcommands are a more powerful versions of previously
```

Comment:
Can we link to the commands? When I read about something new, I'd like to know more about it.

Reply:
And either remove the leading `The` or add `subcommands`.